pages/sending/application.rst

```diff
@@ -0,0 +1,53 @@
+***********************
+Ingest Application Data 
+***********************
+
+We could not cover all languages or examples for many applications in the documentation. But to give a brief overview, this page is created.
+
+
+Ruby on Rails
+=============
+
+This is easy: You just need to combine a few components.
+
+Log all requests and logger calls into Graylog
+-----------------------------------------------
+
+The recommended way to send structured information (i.e. HTTP return code, action, controller, ... in additional fields) about every request and
+explicit ``Rails.logger`` calls is easily accomplished using the `GELF gem <https://rubygems.org/gems/gelf>`__ and
+`lograge <https://github.com/roidrage/lograge>`__. Lograge builds one combined log entry for every request (instead of several lines like the
+standard Rails logger) and has a Graylog output since version 0.2.0.
+
+Start by adding Lograge and the GELF gem to your Gemfile::
+
+  gem "gelf"
+  gem "lograge"
+
+Now configure both in your Rails application. Usually ``config/environments/production.rb`` is a good place for that::
```

Comment:
Is `::` part of the markup syntax, or is it just an accidentally duplicated colon? It's in a few places.

Reply:
That is for the code block needed!

Reply:
Ah! Of course.